server/paymentProviders/transferwise/index.ts

```diff
@@ -139,13 +143,13 @@ async function getRequiredBankInformation(host: any, currency: string): Promise<
     where: { service: 'transferwise', CollectiveId: host.id },
   });
   if (!connectedAccount) {
-    throw new Error('Host is not connected to Transferwise');
+    throw new TransferwiseError('Host is not connected to Transferwise', 'transferwise.error.notconnected');
   }
   await populateProfileId(connectedAccount);
 
   const currencyInfo = find(await getAvailableCurrencies(host), { code: currency });
   if (!currencyInfo) {
-    throw new Error('This currency is not supported');
+    throw new TransferwiseError('This currency is not supported', 'transferwise.error.currencynotsupported');
```

Comment:
Have you considered camelcase to improve readability?

Reply:
Typically, all classes (including errors) are expressed in Pascal case.

Reply:
@kewitz I'm writing about `'transferwise.error.currencynotsupported'`

Reply:
Oh, for sure! I'll update this and the frontend!